src/PostgresWebsockets/Claims.hs

```diff
@@ -43,6 +45,16 @@ validateClaims requestChannel secret jwtToken time =
       Just _ -> throwError "claim is not string value"
       Nothing -> nonExistingClaim defaultVal name
 
+    claimAsJSONList :: Maybe ByteString -> Text -> Claims -> ExceptT Text IO [ByteString]
```

Comment:
we should have additional tests in `ClaimsSpec` covering the new cases (I know the current coverage is not great so feel free to improve on that as well)

Reply:
I have attempted to add some tests. However, I can't run the tests at the moment as I get it to compile